backend/src/main/resources/db-scripts/migration/V1.8__create_document_office_user_group_table.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS
+  documentation_office_user_group (
```

Comment:
Comments for the columns `user_group_path_name` and `is_internal` would be helpful.

Reply:
We have documentation in the DTO entity that corresponds to the table. Should we copy it?
https://github.com/digitalservicebund/ris-backend-service/blob/2d5e1926d623bda2552876abbd36bf91bd71df6c/backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/DocumentationOfficeUserGroupDTO.java#L26-L26

Reply:
Moved to migration (including comments...).